src/main/java/com/bina/lrsim/simulator/ParallelSimulator.java

```diff
@@ -19,13 +20,15 @@
 public class ParallelSimulator {
   private final static Logger log = Logger.getLogger(ParallelSimulator.class.getName());
 
-  public static long process(RandomFragmentGenerator rfg, String outDir, String moviePrefix, String movieSuffix, SamplesDrawer samples, int targetChunk, long targetNumBases, Spec spec, RandomGenerator gen) {
-    Simulator sim = new Simulator(rfg);
+  public static long process(RandomFragmentGenerator randomFragmentGenerator, String outDir, String moviePrefix,
```

Comment:
should not change this, since this is public. Instead, use a new ```Spec``` to specify a new output format with adapter in it